### ISSUE TITLE
Eradicate false positive

### ISSUE THREAD

**Issue by JonathanPlasse** [charliermarsh/ruff#2068]
Hi,
```
# SPDX-License-Identifier: BSD-3-Clause
```
is deleted by eradicate even if it is not a valid code if uncommented.

**Comment by charliermarsh** [comment]
You can configure Ruff to allow those comments via `task-tags`! Check out #2064.

**Comment by JonathanPlasse** [comment]
I could work on it, I seem to remember that there is a helper to check if a python name is valid. As kebab-case is invalid, it would suffice to correct the false positive.

**Comment by JonathanPlasse** [comment]
Kebab-case is not a valid python name so it should not count as valid code.
I do not remember the name of the helper that checks if a python name is valid or not.
It could be used to remove this false positive.
I could work on it if you want.

**Comment by charliermarsh** [comment]
Sure thing!

**Comment by charliermarsh** [comment]
(We have an `is_identifier` method in `src/python/identifiers.rs`.)

**Comment by JonathanPlasse** [comment]
It is not possible to fix.
In the context of a `dict`. This is a valid code if `SPDX`, `License`, `Identifier`, `BSD`, and `Clause` are `int` variables.
```python
{
    SPDX-License-Identifier: BSD-3-Clause
}
```

**Comment by charliermarsh** [comment]
Ah ok. Thanks for taking a look. At least we have a valid workaround.

**Comment by JonathanPlasse** [comment]
You are welcome.
